src/main/java/org/orecruncher/dsurround/lib/world/WorldUtils.java

```diff
@@ -38,7 +38,7 @@ public static BlockPos getTopSolidOrLiquidBlock(final World world, final BlockPo
     }
 
     public static float getTemperatureAt(final World world, final BlockPos pos) {
-        return world.getBiomeAccess().getBiome(pos).getTemperature(pos);
+        return world.getBiomeAccess().getBiome(pos).getTemperature();
```

Comment:
Are the semantics here the same?

Reply:
Yes, the old method was removed in fact.